src/Components/Btn.jsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```

Comment:
This component should not exists, whats the purpose of wraping a component button inside another component button?

Reply:
Btn component removed.